.github/pull_request_template.md

```diff
@@ -25,11 +25,12 @@ If this PR fixes a COPS ticket, include it after the note like: "CLI: Some bug f
 
 ```
 
-**If the PR adds a version bump, does it add a breaking change in License**:
+**Checklist**
 <!--
 For example, If a chart changes license from say Apache License to GNU AFFERO GENERAL PUBLIC LICENSE then
 that would have legal repercussions (as we ship helm charts, image bundles for airgapped etc.,) and multiple
 parties (Like Product, Legal for example) need to be notified when such a change happens.
 -->
 
-- [ ] No License Change (or NA).
+- [ ] If the PR adds a version bump, ensure there is no breaking change in Licensing model (or NA).
+- [ ] If a chart is changed or app configuration is signficantly changed, the chart version is correctly incremented (so that apps are not automatically upgraded from a previous version of DKP).
```

Comment:
GH hasn't been letting me suggest edits today :/
minor type `significantly`

Reply:
```suggestion
- [ ] If a chart is changed or app configuration is significantly changed, the chart version is correctly incremented (so that apps are not automatically upgraded from a previous version of DKP).
```
yay suggestions are finally working for me again